diff --git a/src/render/ci.rs b/src/render/ci.rs
--- a/src/render/ci.rs
+++ b/src/render/ci.rs
@@ -3388,1 +3388,1 @@
-        "  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref_name }}\n",
+        "  group: ${{ github.workflow }}-${{ github.event.pull_request.head.ref || github.ref_name }}\n",
diff --git a/tests/init_ci.rs b/tests/init_ci.rs
--- a/tests/init_ci.rs
+++ b/tests/init_ci.rs
@@ -499,1 +499,1 @@
-            .contains("group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref_name }}")
+            .contains("group: ${{ github.workflow }}-${{ github.event.pull_request.head.ref || github.ref_name }}")
@@ -946,1 +946,1 @@
-            .contains("group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref_name }}")
+            .contains("group: ${{ github.workflow }}-${{ github.event.pull_request.head.ref || github.ref_name }}")
@@ -3623,1 +3623,1 @@
-        "group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref_name }}"
+        "group: ${{ github.workflow }}-${{ github.event.pull_request.head.ref || github.ref_name }}"
